client/src/search/EverythingSearch.js

```diff
@@ -258,27 +258,25 @@ const EverythingSearch = withRouter(
       );
 
       return (
-        <div>
-          <div className="col-md-12">
-            <Typeahead
-              placeholder={placeholder}
-              search_configs={this.get_search_configs()}
-              utility_buttons={
-                !hide_utility_buttons && [
-                  <SearchOptions
-                    key="SearchOptions"
-                    option_checkboxes={option_checkboxes}
-                  />,
-                ]
-              }
-              page_size={page_size}
-              on_query={on_query}
-              on_select={this.onSelect}
-              additional_a11y_description={text_maker(
-                "everything_search_additional_a11y_description"
-              )}
-            />
-          </div>
+        <div className="col-md-12">
```

Comment:
Switch to `fcol`s. I'm going to stop commenting on these if I see others, something we should try and do a general sweep for.

Reply:
Oh, hahaha, this was the last file in the diff XD